README.md

```diff
@@ -1,44 +1,128 @@
 # Directed Acyclic Workflow Graph Scheduling
 
+Reproducible research and reusable acyclic workflows in Python. Execute code on HPC systems as if you executed them on your machine!
+
+## Motivation
+
+Would you like fully reproducible research or reusable workflows that seamlessly run on HPC clusters?
+Tired of writing and managing large Slurm submission scripts? Do you have comment out large parts of your pipeline whenever its results have been generated? Hate YAML?
+Don't waste your precious time! `dawgz` allows you to directly describe complex pipelines in Python, that run on your personal computer and large HPC clusters.
+
```

Comment:
# Directed Acyclic Workflow Graph Scheduling

Would you like fully reproducible and reusable workflows that run on HPC clusters as seamlessly as on your machine? Tired of writing and managing large Slurm submission scripts? Do you have to comment out large parts of your pipeline whenever something failed? Hate YAML and config files? Then `dawgz` is made for you!

The `dawgz` package allows you to define and execute complex workflows, directly in Python. It provides a lightweight interface that enables to define jobs along with their dependencies, requirements, postconditions, etc. After defining the workflow, you can schedule target jobs in a single line of code and all their dependencies will be automatically scheduled as well. Importantly, the scheduling backend can be changed with a single parameter, which enables to execute the same workflow on your personal computer and large HPC clusters.

> `dawgz` should be pronounced *dogs* 🐶